scripts/superdesk-search/react.js

```diff
@@ -653,12 +644,12 @@
                                     React.createElement('span', {className: 'broadcast-status', tooltip: broadcast.status}, '!') :
                                     null,
                                 flags.marked_for_not_publication ?
-                                    React.createElement('div', {className: 'state-label not-for-publication'},
-                                        gettext('Not For Publication')) :
+                                    React.createElement('div', {className: 'state-label not-for-publication',
+                                        title: gettext('not for publication')}, gettext('Not For Publications')) :
```

Comment:
would be better to have same text for both text and title, so there is only 1 string to translate, not 2